package.json

```diff
@@ -16,20 +33,23 @@
     "deploy": "f () { npm run build:travis && npm run build:local && lerna exec --scope @deriv/core -- npm run deploy $@ ;}; f",
     "deploy:clean": "f () { npm run build:travis && npm run build:local && lerna exec --scope @deriv/core -- npm run deploy:clean $@ ;}; f",
     "deploy:folder": "f () { npm run build:travis && npm run build:local $1 && lerna exec --scope @deriv/core -- npm run deploy:folder $@ ;}; f",
-    "prettify": "f () { lerna exec --scope=@deriv/${1:-'*'} -- npm run --silent prettify ;}; f",
+    "prettify": "prettier --config ./.prettierrc --write \"./packages/*/{stories,script,src,build}/**/*.{js,jsx,scss}\"",
+    "precommit": "lint-staged --allow-empty",
     "publish:p2p": "f () { lerna publish from-package ;}; f",
     "serve": "f () { lerna exec --scope=@deriv/${1:-'*'} -- npm run serve $npm_config_open;}; f",
     "start": "f () { lerna exec --scope=@deriv/${1:-'*'} -- npm run start ;}; f",
-    "test": "f () { bash ./scripts/circleci-config.test.sh && lerna exec --scope=@deriv/${1:-'*'} --parallel -- npm run test ;}; f",
+    "test": "f () { bash ./scripts/circleci-config.test.sh && npm run test:stylelint  && eslint \"./packages/*/src/**/*.?(js|jsx)\" && lerna exec --scope=@deriv/${1:-'*'} --parallel -- npm run test ;}; f",
     "test:eslint": "f () { lerna exec --scope=@deriv/${1:-'*'} -- npm run test:eslint ;}; f",
     "test:mocha": "f () { lerna exec --scope=@deriv/${1:-'*'} -- npm run test:mocha ;}; f",
-    "test:stylelint": "f () { lerna exec --scope=@deriv/${1:-'*'} -- npm run test:stylelint ;}; f",
-    "translate": "f () { lerna exec --scope @deriv/translations -- npm run translate ;}; f"
+    "test:stylelint": "stylelint \"./packages/{account/src/Styles,components/src/components,(trader|core)/src/sass,bot-web-ui/src/assets/sass,p2p/src/components,shared/src/styles}/**/*.s?(a|c)ss\"",
```

Comment:
any way to run it on every package inside `/packages`? so we don't need to remember to add/remove packages here

Reply:
```
stylelint \"./packages/*/src/**/*.s(a|c)ss\"
```
this could work

Reply:
@oskar-binary No, unless we want to install it locally? :thinking: 

@balakrishna-binary Yes, updated. thanks man! :)